yt_dlp/extractor/polsatgo.py

```diff
@@ -13,22 +13,54 @@
 class PolsatGoIE(InfoExtractor):
     _VALID_URL = r'https?://(?:www\.)?polsat(?:box)?go\.pl/.+/(?P<id>[0-9a-fA-F]+)(?:[/#?]|$)'
     _TESTS = [{
-        'url': 'https://polsatgo.pl/wideo/seriale/swiat-wedlug-kiepskich/5024045/sezon-1/5028300/swiat-wedlug-kiepskich-odcinek-88/4121',
+        'url': 'https://polsatgo.pl/wideo/seriale/swiat-wedlug-kiepskich/5024025/sezon-1/5024197/swiat-wedlug-kiepskich-odcinek-1/4208/ogladaj',
         'info_dict': {
-            'id': '4121',
+            'id': '4208',
             'ext': 'mp4',
-            'title': 'Świat według Kiepskich - Odcinek 88',
+            'title': 'Świat według Kiepskich - Odcinek 1',
             'age_limit': 12,
         },
+        'params': {
+            'listsubtitles': True,
+        },
```

Comment:
?

Reply:
otherwise `_get_subtitles` doesn't get called

Reply:
but subtitles are not being tested here. Add a `'subtitles': 'count:xxx'` to test